_data/locales/pt.yml

```diff
@@ -19,8 +19,8 @@ pt:
       install: Instale o Homebrew
       paste: Cole este código numa linha de comandos do seu Terminal macOS ou Linux.
       what: O script explica o que irá fazer e faz uma pausa antes de ser executado. Leia sobre outras <a href="https://docs.brew.sh/Installation">opções de instalação</a>.
-      pkginstaller: If you're on macOS, try our new <code>.pkg</code> installer.
-      githubrelease: Download it from <a href="https://github.com/Homebrew/brew/releases/latest">Homebrew's latest GitHub release</a>.
+      pkginstaller: Se tu és em macOS, tenta o novo instalador do .pkg<code>.pkg</code>.
```

Comment:
```suggestion
      pkginstaller: Se tu és em macOS, tenta o novo instalador do <code>.pkg</code>.
```

Reply:
pkg is already written down in the `<code> </code>` block